app/styles/quickgo.scss

```diff
@@ -194,6 +194,11 @@ span.has-tip{
 }
 
 
+#morePanel div.row{
```

Comment:
We need to use the framework instead of doing component-based fixes. Reading the Foundation docs, there's a class called `.expanded` which allows full-screen width instead of the default 1200 pixels:

http://foundation.zurb.com/sites/docs/grid.html#fluid-row

Looking back at how full-width pages was implemented, this is what we should be using instead. So please remove all code relating to `fullWidthPage` and use `.expanded` with `.row` on pages for which the content should be full width.